### ISSUE TITLE
Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by felixbarny** [jzillmann/jmh-visualizer#20]
Hi and thanks for your awesome project!

I think I have encountered a bug when analyzing some of my benchmark results:

http://jmh.morethan.io/?sources=https://gist.githubusercontent.com/felixbarny/34dbb685a63ca56c55ec3d2dc5177151/raw/4e57b0c59505c5bd84410213af90039adbfd6350/synchronized-map.json

When clicking on the magnifying glass, I get a blank page and this error in the console in the latest version of Chrome:

```
bundle.js:formatted:19990 TypeError: Cannot read property 'forEach' of undefined
    at t.value (bundle.js:formatted:65932)
    at bundle.js:formatted:64463
    at Array.forEach (<anonymous>)
    at bundle.js:formatted:64459
    at Array.map (<anonymous>)
    at Ex (bundle.js:formatted:64454)
    at bundle.js:formatted:64805
    at t.value (bundle.js:formatted:64810)
    at Co (bundle.js:formatted:19527)
    at Bo (bundle.js:formatted:19724)
Uo @ bundle.js:formatted:19990
Vo.n.callback @ bundle.js:formatted:20206
Wa @ bundle.js:formatted:18998
Ha @ bundle.js:formatted:19014
$i @ bundle.js:formatted:20907
Xi @ bundle.js:formatted:20656
Vi @ bundle.js:formatted:20615
Ki @ bundle.js:formatted:20603
ns @ bundle.js:formatted:21007
In @ bundle.js:formatted:17753
```

**Comment by jzillmann** [comment]
Hey @felixbarny thanks for reporting, i will look into it sooooon!

**Comment by jzillmann** [comment]
@felixbarny Should be fixed now (release 0.9.2), please report back if you have any problems!
